civiclens/utils/text.py

```diff
@@ -0,0 +1,57 @@
+import re
+
+
+def clean_text(text: str, patterns: list[tuple] = []) -> str:
```

Comment:
is patterns intended to be actually used? If so, could use a definition in the documentation

Reply:
sorry i left this out of the docstring, the three fixed regex statements were things I came across when processing text, and the patterns can allow someone else to input additional cleaning statements.

its not a perfect function, i'll admit